index_mobile.css

```diff
@@ -0,0 +1,9 @@
+.flex,
+.flex > *,
+.flex__middle--inner,
+.flex__bottom {
```

Comment:
Alternatively we could center everything absolutely on the page using `position: absolute` on the body. It would take a few additional properties to configure also.
```
  body {
    position: absolute;
    top: 50%;
    left: 50%;
    transform: translate(-50%,-50%);
    }
```
![image](https://user-images.githubusercontent.com/33106391/227398594-b2dd3e6e-60ec-4318-b095-62861ff3e072.png)

Reply:
Hey @mnichols08 ! Thanks a lot for taking the time to go over my code, I really appreciate it (and no worries, I am not disapointed, I am learning a lot from you guys as we discussed yesterday with @PaulaR-05  ;) ). Putting a` text-align: center` is actually what I did yesterday after my talk with Gabriela and it was all peachy... so I will go over the code and your notes again to try to resolve this this morning.
![image](https://user-images.githubusercontent.com/90863470/227543597-faa6ffa2-09d6-4918-9887-986211e29a5d.png)